.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "lib/forge-std"]
+	path = lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
```

Comment:
In general if we add `dss-test`, we remove `forge-std`.

Reply:
Done